build.xml

```diff
@@ -35,13 +35,32 @@
 
   <target name="codestyle" depends="prepare">
     <phpcodesniffer standard="codingstyle.xml">
+      <formatter type="full" usefile="false"/>
       <formatter type="checkstyle" outfile="${builddir}/checkstyle.xml"/>
-      <fileset dir="features">
+      <fileset dir="centreon-build">
+        <include name="**/*.php"/>
+      </fileset>
+      <fileset dir="doc">
+        <include name="**/*.php"/>
+      </fileset>
+      <fileset dir="GPL_LIB">
+        <include name="**/*.php"/>
+      </fileset>
+      <fileset dir="node_modules">
         <include name="**/*.php"/>
       </fileset>
       <fileset dir="tests">
         <include name="**/*.php"/>
       </fileset>
+      <fileset dir="tmpl">
+        <include name="**/*.php"/>
+      </fileset>
+      <fileset dir="vendor">
+        <include name="**/*.php"/>
```

Comment:
We can't control external packages that's why there is no sense to check package dependencies for their code style, right?

Reply:
first : vendor is a dev folder, which partially dupicate centreon-build. So no need to check it.
second : indeed 99.9% of vendor is external, so we shouldn't (coding-style or not) modify these files ;)